Findologic/Search/Controller/Export/ExportController.php

```diff
@@ -104,8 +104,16 @@ public function execute()
                 ->setContents('Run "composer require findologic/libflexport" in your project directory');
         }
 
-        $xmlExample = new XmlExample();
-        $xml = $xmlExample->createExport();
+        if (!file_exists(__DIR__ . '/../../../Export/Export.php')) {
+            // TODO: Add the link to the export plugin.
+            return $this->rawResponse
+                ->setHeader('Content-type', 'text/plain')
+                ->setContents('Please install the export plugin. You can find it here http://something');
+        }
+
+        require_once __DIR__ . '/../../../Export/Export.php';
```

Comment:
Is the base plugin and the export plugin in one Findologic folder?

Reply:
Yes in Magento 2 it is per-vendor.